test/suite/test_checkpoint10.py

```diff
@@ -38,6 +38,7 @@
 # Test what happens if we create an inconsistent checkpoint and then try to
 # open it for read. No timestamps in this version.
 
+@wttest.skip_for_hook("tiered", "FIXME-WT-9809 - Fails for tiered")
```

Comment:
These checkpoint decorators are a bit interesting. We previously filtered on `test_checkpoint.test_checkpoint` in `hook_tiered.py`, and most of our checkpoint tests (excluding `test_checkpoint02` - `09`) don't have the test number in their class or test function. As a result we filter a large fraction but not all of the checkpoint tests.

Since this PR is a refactor I didn't want to change up which tests get run, hence the decorators on a subset of checkpoint tests.